src/context.ts

```diff
@@ -120,4 +127,9 @@ export class Context implements Required<ContextValue> {
     if (this.ctx.simulation) return this.ctx.simulation;
     throw new SdkError("simulation configuration must be defined in Context for this feature to work.");
   }
+
+  get cache(): CacheConfiguration {
+    if (this.ctx.cache) return this.ctx.cache;
```

Comment:
do our lint rules allow this (inline condition)? does this project use lint?

Reply:
Yes we have a lint script, this is ok according to it